src/test/kotlin/com/autonomousapps/internal/parse/KotlinBuildScriptDependenciesRewriterTest.kt

```diff
@@ -485,6 +485,45 @@ internal class KotlinBuildScriptDependenciesRewriterTest {
     )
   }
 
+  @Test fun `can parse complex dependencies`() {
```

Comment:
dependencies in `if` statement and non-dependency declarations can be parsed but will not be rewritten